### ISSUE TITLE
Maybe a command to packing specific audio track/sub into the mkv ?

### ISSUE THREAD

**Issue by lunaticwhat** [ToaHartor/GI-cutscenes#10]
It's much better if there is a command for it. I can fork the repo and take a look at making some changes later ;)

**Comment by ToaHartor** [comment]
Sure ! That's a good improvement to the CLI, so if you want to work on it, feel free to do so and make a pull request. 

Then, I guess the user would be able to list the language codes they want to merge (default to all languages). For the subs part, maybe there would be some tricks to do with the `--subs` option.

**Comment by ToaHartor** [comment]
Oh, while I think about this,  it can also be done at the demuxing stage for the audio tracks, only choosing which track is extracted.

**Comment by lunaticwhat** [comment]
> Oh, while I think about this, it can also be done at the demuxing stage for the audio tracks, only choosing which track is extracted.

Yeah that is a good idea, I was trying to messing with mkv repack process to select specific audio tracks/subs but seems I have so many _skill issue_ to have it done :sadge:
